### DISCUSSION TITLE
Feature suggestion: Rumble

### DISCUSSION THREAD

**Discussion by Crilum** [cookpete/react-player#1777] [Ideas]
[Rumble](https://rumble.com) would be a good addition to the list of players.

**Comment by brentarias** [comment]
I second,  Rumble is youtube's biggest competitor

**Comment by EHxuban11** [comment]
Hello, I need Rumble too for a project. Is there any news about this subject? I've tried rumble on the Reac player demo page (https://cookpete.github.io/react-player/) and it does not work.

**Comment by TrySpace** [comment]
Should be doable with this API: https://www.rumbleplayer.com/developers/Player-Methods.html
Looks fairly similar to: https://github.com/cookpete/react-player/blob/master/src/players/YouTube.js
